OoyalaSkinSampleApp/OoyalaSkinSampleApp/Players/DefaultSkinPlayerViewController.m

```diff
@@ -63,6 +68,16 @@ - (void)viewDidLoad {
                                            selector:@selector(notificationHandler:)
                                                name:nil
                                              object:self.skinController];
+  [[NSNotificationCenter defaultCenter] addObserver: self
+                                           selector:@selector(notificationHandler:)
+                                               name:nil
+                                             object:ooyalaPlayer];
+  
+  // In QA Mode , making textView visible
+  if(self.qaModeEnabled==YES){
```

Comment:
@dchauOoyala, to your last question, `YES` and `NO` are preferred over `true` and `false` in ObjC. You'll see it like that everywhere, at the end they're aliases to `true` and `false`.